WebApp/static/core/js/views/tabs/Main.js

```diff
@@ -2371,3 +2371,14 @@ Ext.define('Ext.dirac.views.tabs.Main', {
       }
 
     });
+  /*
+   * Welcome window
+   */
+ // if (GLOBAL.APP.configData.user.username) {
+ //   Ext.Msg.alert(GLOBAL.APP.configData["user"]["username"] + ', welcome to DIRAC!', "<b>D</b>istributed <b>I</b>nfrastructure with <b>R</b>emote <b>A</b>gent <b>C</b>ontrol INTERWARE is a software framework for distributed computing providing a complete solution to one (or more) user community requiring access to distributed resources. DIRAC builds a layer between the users and the resources offering a common interface to a number of heterogeneous providers, integrating them in a seamless manner, providing interoperability, at the same time as an optimized, transparent and reliable usage of the resources. <br> <a href='http://diracgrid.org'>Discover more...</a>" );
+ // } else {
+    /*
+     * If the user is not registered
+     */
+  //  Ext.Msg.alert('Welcome to DIRAC!', "Distributed Infrastructure with Remote Agent Control INTERWARE is a software framework for distributed computing providing a complete solution to one (or more) user community requiring access to distributed resources. DIRAC builds a layer between the users and the resources offering a common interface to a number of heterogeneous providers, integrating them in a seamless manner, providing interoperability, at the same time as an optimized, transparent and reliable usage of the resources. <br> <i><a href='http://diracgrid.org'>Discover more...</a></i> <br><br> <b>You are not registry user.</b> <br> If you want to use a user proxy, we assume that you already have a user certificate, so in this case create a directory .globus in your home directory and copy the certificate files (public and pruvate keys in pem format) to this directory: <br> $ mkdir ~/.globus <br> $ cp <<certificate files>> ~/.globus/ <br> At this point you need a proxy, you should do: <br> $ dirac-proxy-init");
+  //}
```

Comment:
Could you please remove the commented lines? Please add this line Ext.create('Ext.dirac.views.desktop.WelcomeWindow'); also to the Ext.dirac.views.tabs.Main